scripts/filestash/filestash_config.json.template

```diff
@@ -0,0 +1,90 @@
+{
+  "general": {
+      "name": null,
+      "port": null,
+      "host": null,
+      "secret_key": "zUMyrLIDRMJKEdWw",
+      "force_ssl": null,
+      "editor": null,
+      "fork_button": null,
+      "logout": null,
+      "display_hidden": null,
+      "refresh_after_upload": null,
+      "auto_connect": null,
+      "upload_button": null,
+      "upload_pool_size": null,
+      "filepage_default_view": "list",
+      "filepage_default_sort": "date",
+      "cookie_timeout": null,
+      "custom_css": null
+  },
+  "features": {
+      "share": {
+          "enable": null,
+          "default_access": null,
+          "redirect": null
+      },
+      "protection": {
+          "zip_timeout": null,
+          "enable": null,
+          "disable_svg": null
+      },
+      "office": {
+          "enable": null,
+          "onlyoffice_server": null
+      },
+      "server": {
+          "console_enable": null,
+          "tor_enable": null,
+          "tor_url": null
+      },
+      "syncthing": {
+          "enable": null,
+          "server_url": null
+      },
+      "image": {
+          "enable_image": null,
+          "thumbnail_size": null,
+          "thumbnail_quality": null,
+          "thumbnail_caching": null,
+          "image_quality": null,
+          "image_caching": null
+      },
+      "search": {
+          "explore_timeout": null
+      },
+      "video": {
+          "blacklist_format": null,
+          "enable_transcoder": null
+      }
+  },
+  "log": {
+      "enable": null,
+      "level": null,
+      "telemetry": null
+  },
+  "email": {
+      "server": null,
+      "port": null,
+      "username": null,
+      "password": null,
+      "from": null
+  },
+  "auth": {
+      "admin": "$$2a$$10$$viR16hXd35bAaEJFEgpd9OqIzNBb/VoIsgQ8P3SjKxolpEQEHltrW"
```

Comment:
How is this generated? I'd like to auto-generated in the ops-stack

Reply:
@GitHK

Reply:
Looks just like this https://docs.docker.com/registry/deploying/#native-basic-auth

Reply:
I'm sorry it has nothing to do with this. It was generated by the program.
When you first start it it will ask for admin credentials. I've only escaped the `$` signs in there.